### ISSUE TITLE
Show number of skipped tests

### ISSUE THREAD

**Issue by jimbuck** [zoubin/tap-summary#3]
`TAP` states that `SKIP` tests should not be counted as run tests. Is this something we can display at the end (just the count, or optionally the titles)? If yes I'd be happy to work on it myself.

**Comment by jimbuck** [comment]
Looks like the work might have to [start here first](https://github.com/scottcorgan/tap-out/issues/16).

**Comment by zoubin** [comment]
>  Is this something we can display at the end (just the count, or optionally the titles)?

Yes, that would be nice!
We can add a line showing the count in the summary section, and an additional section to display the titles.
I am happy to see that done 😄

**Comment by RReverser** [comment]
@JimmyBoh 

> If yes I'd be happy to work on it myself.

Have you had any chance to work on it?

**Comment by jimbuck** [comment]
@RReverser No, not yet. I got distracted by learning how to make custom yeoman generators... :smile:
